src/main/webapp/intrigue-api/graphql.js

```diff
@@ -381,6 +381,24 @@ const Query = {
   facet,
 }
 
+const btoa = arg => {
+  if (typeof window !== 'undefined') {
+    return window.btoa(arg)
+  }
+  return Buffer.from(arg).toString('base64')
+}
+
+const logIn = async (parent, args, { fetch }) => {
+  const { username, password } = args
+  const authToken = btoa(`${username}:${password}`)
```

Comment:
Don't you need to include the method of auth as well? `Authorization: Basic ${auth}`